### PR TITLE
initial pull request to support running Atlas Tests in OHDSI jenkins 

### PR DESCRIPTION
**Pull request by leeevans**
see commit messages for details.  For your testing add the new data property 'environmentname' and set it to the value ARACHNE in application.properties.

### CODE REVIEW

File: src/test/java/atlastests/LoginStepsDefs.java [hunk 1]
```diff
@@ -22,8 +22,8 @@ public static void clickToLogInLink() {
     }
 
     @When("^click QA Arachne$")
-    public static void clickTopQAArachne() {
-        $(By.xpath("//*[@class='paddedWrapper']/div/span")).shouldHave(text("ARACHNE")).click();
+    public static void clickTopQAArachne() throws Exception {
```
Comment (KSadomtsev) [inline src/test/java/atlastests/LoginStepsDefs.java]: Methods inside don't throw execeptions, i think we don't need this

Comment (KSadomtsev) [inline src/test/java/atlastests/LoginStepsDefs.java:25]: i think we should rename this method after your changes, also in annotation and .feature file.

**Review by KSadomtsev — CHANGES_REQUESTED** [review]

**Comment by leeevans** [comment]
I squashed the 'added missing exception' commit into the 'added environmentname data property 'commit - they should have been a single commit in the first place - please review again

File: src/test/java/atlastests/LoginStepsDefs.java (hunk 1 above)
Comment (leeevans) [reply src/test/java/atlastests/LoginStepsDefs.java:25]: agreed

**Comment by KSadomtsev** [comment]
> I squashed the 'added missing exception' commit into the 'added environmentname data property 'commit - they should have been a single commit in the first place - please review again

i did. but i see here naming problem, i mean it would be better to change method name to clickToEnvironmentName for example

**Pull request merged**